llama-index-integrations/vector_stores/llama-index-vector-stores-postgres/llama_index/vector_stores/postgres/base.py

```diff
@@ -169,6 +171,7 @@ def __init__(
         perform_setup: bool = True,
         debug: bool = False,
         use_jsonb: bool = False,
+        hnsw_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
Can we include this in the docstring? Even some small explanation of the typical parameters 🤔 

It might also be good to update `docs/docs/examples/vector_stores/postgres.ipynb` to use this by default (imo most people should probably be using hnsw if they want the best performance)